raven/README.md

```diff
@@ -42,6 +42,14 @@ Multiple values of the permission granting service resourceID(s) may be provided
 settings. All services will be consulted but a single positive result is sufficient to grant permission for an 
 action.
 
+Another set of optional keys configure raven to prioritize sites returned in a transfer, with higher priority
+sites first in the list of transfer URL's to GET or PUT a file.
```

Comment:
The config options below only apply to PUT, but the above says GET or PUT.

The StorageSite class has a "name" attribute (it's just part of the resourceID and I'm kind of regretting it already) that might cause confusion with `{storage site name}` below. Maybe we should should use `{rule name}` (or `{entry name}` like in baldur)? 

Aside: over in baldur, we put the `{entry name} {regular expression}` on the same line, where we should have used 
```
{entry name}.pattern = {regular expression}
```
to re-use existing parsing functionality like here. That's one of those small style inconsistencies :-)

If we had done that, we would have noticed that the {pattern}'s in baldur and the {namespace}'s here are essentially **the same thing done two slightly different ways. :-(**

Also, the example below uses an @ sign which I know is just another character to be read as "at CADC", but I think it might be confusing and look like it's important when it isn't.

Reply:
I updated the README to use `entry name` to match baldur. And removed the `@` before `CADC` in the example usage because yes one might wonder if it has some hidden meaning.